zerver/models.py

```diff
@@ -1506,14 +1506,26 @@ def __str__(self) -> str:
         return "<%s: %s / %s / %s>" % (self.__class__.__name__, display_recipient,
                                        self.subject, self.sender)
 
+class ArchiveTransaction(models.Model):
+    archive_timestamp = models.DateTimeField(default=timezone_now, db_index=True)  # type: datetime.datetime
```

Comment:
This should probably just be called `timestamp`

Reply:
Fixed by changing the first commit